### PR TITLE
Fixes #4218

### LINKED ISSUE #4218: Derelict mommis
Derelict Mommins need better check.

As of right now they can build a mass driver or conveyer belt line+jump into a locker to get off the Derelict.

Not only that you can do this with the nuke disk by trapping it at the end of the Z level and NO one can grab it.

### PR DESCRIPTION
**Pull request by PJB3005**
Now using recursive type checks!

### CODE REVIEW

File: code/game/turfs/space/space.dm [hunk 1]
```diff
@@ -52,17 +52,18 @@
...
 				return
 
-			var/disks_list = recursive_type_check(A, /obj/item/weapon/disk/nuclear)
+			var/specials_list = recursive_type_check(A, list(/obj/item/weapon/disk/nuclear, /mob/living/silicon/robot/mommi))
 
 			if(istype(A, /obj/structure/stool/bed/chair/vehicle))
 				var/obj/structure/stool/bed/chair/vehicle/B = A
 				if(B.buckled_mob)
-					disks_list = recursive_type_check(B.buckled_mob, /obj/item/weapon/disk/nuclear)
+					specials_list = recursive_type_check(B.buckled_mob, /obj/item/weapon/disk/nuclear)
```
Comment (ComicIronic) [inline code/game/turfs/space/space.dm]: Shouldn't this also check if B is a MoMMI?

**Comment by clusterfack** [comment]
This is a really small change please squash

**Comment by d3athrow** [comment]
how the fuck did this not get auto closed

**Comment by d3athrow** [comment]
Merged with 3583b2c1e2962a5a751f260aad0af50791e3c56d

**Pull request closed without merging**